doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,49 @@
+## Using YUM repositories
+
+There are 2 available packages:
```

Comment:
A short explanation should be added in each case, eg.

* *nighly*, for RPMs which are built from master branch each nigth (at {hour}).
    * Doubt: do you have only one RPM (updated each nigth with the new date)? Or all the RPM generated each nigth? (In the later case I guess that you would have some kind of time window, otherwise the disk in your repository machine will exhaust at some moment)
* *release*, for "official" release RPMs. There isn't a fixed released period although it use to be aroung a release each 1-2 months.

Please expand or modify the above text as you wish.

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9